_includes/accessibility-tests/a11y-note.html

```diff
@@ -1,13 +1,15 @@
+{% assign component_name = page.title | downcase %}
+
 <div class="usa-alert usa-alert--info usa-alert--no-icon measure-5 margin-bottom-2">
   <div class="usa-alert__body">
     <p class="usa-alert__text text-bold">
-      Test the {{ componentName }} in your own project.
+      Test the {{ component_name }} component in your own project.
```

Comment:
`componentName` is used elsewhere, which might cause confusion. 

Do we have an issue to make that consistent?